core/src/main/java/io/questdb/griffin/engine/SystemOperator.java

```diff
@@ -34,27 +34,73 @@
 import io.questdb.griffin.QueryRegistry;
 import io.questdb.griffin.SqlException;
 import io.questdb.griffin.SqlExecutionContext;
+import io.questdb.log.Log;
+import io.questdb.log.LogFactory;
 import io.questdb.mp.SCSequence;
 import io.questdb.std.Chars;
 
 // Factory that adds query to registry on getCursor() and removes on cursor close().
-public class RegisteredRecordCursorFactory extends AbstractRecordCursorFactory {
-
+public class SystemOperator extends AbstractRecordCursorFactory {
+    private static final Log LOG = LogFactory.getLog(SystemOperator.class);
```

Comment:
I don't like that all queries will be logged with `SystemOperator` logger. It is a very vague name.
Maybe `QueryProgress` would be better.

Reply:
renamed in [d70de10](https://github.com/questdb/questdb/pull/4532/commits/d70de101cb06100c7dbd7403f6a851f6631ad9f5)